src/main/java/app/controllers/SetSprintDataController.java

```diff
@@ -106,8 +106,30 @@ private void clearFields() {
         dataFimPicker.setValue(null);
     }
 
+    @FXML
+    private void deleteSprint() {
+        Utils.setAlert("WARNING", "Deleção da sprint", "Tem certeza que deseja deletá-la?",() -> {
+        int sprintId = 0;
+        ObservableList<SprintModel> selectedItems = tableData.getSelectionModel().getSelectedItems();
+        for (SprintModel sprint : selectedItems) {
```

Comment:
ao inves de for, usa um selectedItems.get(0) ou algo assim, para pegar o primeiro item dalista